taar/recommenders/local_recommender.py

```diff
@@ -0,0 +1,51 @@
+import requests
+
+ADDON_LIST_PER_LOCALE_URL =\
```

Comment:
If you change the behaviour of fetch_json as described below, we'd be able to merge this PR even without having the correct URL here for now ;) It would simply fail and skip the locale recommendation!